crates/viewer/re_viewer/src/web.rs

```diff
@@ -316,6 +316,7 @@ pub struct AppOptions {
     url: Option<StringOrStringArray>,
     manifest_url: Option<String>,
     render_backend: Option<String>,
+    video_decoder: Option<String>,
```

Comment:
The docstring for this `struct` reads:

> // Keep in sync with the `AppOptions` interface in `rerun_js/web-viewer/index.ts`.

Yet there are no changes to the file in this PR

Reply:
oof, bad miss there. I wonder how we test that rerun_js binding actually 😬